Update PyTorch for XNNPACK 87ee0b4

GregoryComer opened the pull request:
Summary: Update XNNPACK library version.

Test Plan: Combined diff CI is clean: D61586079 (all changes, has to be split out for export).

Differential Revision: D61822610

mcr229 reviewed (approved):
will wait on CI, but stamping for now.

GregoryComer committed:
Update PyTorch for XNNPACK 87ee0b4 (#134518)

Summary:
Update XNNPACK library version.

Pull Request resolved: https://github.com/pytorch/pytorch/pull/134518

Test Plan: Combined diff CI is clean: D61822606 (all changes, has to be split out for export).

Reviewed By: digantdesai, mcr229

Differential Revision: D61822610

Pulled By: GregoryComer
